### PR TITLE
Added trailhead.salesforce.com customizations

### PR DESCRIPTION
**Pull request by georgekarapi**

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -10209,6 +10209,30 @@ CSS
...
 ================================
 
+trailhead.salesforce.com
+
+INVERT
+lwc-tds-header
+
+CSS
+.search-input__input {
+    background-color: var(--darkreader-neutral-background);
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:10219]: Don't forget to put `!important;`

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -10209,6 +10209,30 @@ CSS
...
+}
+.user-info__name {
+    color: inherit !important;                                                                              
+}
+.progress-ring__content {
+    background-color: white !important;
+}
+[data-layout="desktop"] .nav-list-item__link:hover {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:10227]: I suggest having something like
```CSS
[data-layout="desktop"] .nav-list-item__link:hover,
[data-layout="desktop"] .nav-list-item__button[aria-expanded="true"] {
    background: inherit !important;
}
```

**Review by Gusted — CHANGES_REQUESTED** [review]

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (georgekarapi) [reply src/config/dynamic-theme-fixes.config:10227]: Great point!

**Review by Gusted — APPROVED** [review]
Thank you!

**Pull request merged**